include/exiv2/value.hpp

```diff
@@ -1270,16 +1270,28 @@ class ValueType : public Value {
     }
 
     // Check for integer overflow.
+#if __cplusplus >= 201703L
```

Comment:
[question] Does it make sense to add all these conditional code? We are already indicating in the CMake code that we need a C++17 compiler for this project. Therefore, I do not see the point of having conditionals for older versions of the C++ standard.

Reply:
This is for the headers. It was requested earlier that they be kept compatible with C++11.